cloud-config/run-scripts/app-encd.sh

```diff
@@ -38,7 +38,7 @@ if [ ! -f "$bin_build_path" ]; then
 fi
 
 # Run bin/buildout
-bin_build_cmd="$ENCD_HOME/bin/buildout -c $ENCD_ROLE.cfg buildout:es-ip=$ENCD_ES_IP buildout:es-port=$ENCD_ES_PORT buildout:pg-uri=$PG_URI buildout:fe-ip=$ENCD_FE_IP buildout:remote_indexing=$ENCD_REMOTE_INDEXING buildout:index_procs=$ENCD_INDEX_PROCS"
+bin_build_cmd="$ENCD_HOME/bin/buildout -c $ENCD_ROLE.cfg buildout:es-ip=$ENCD_ES_IP buildout:es-port=$ENCD_ES_PORT buildout:pg-uri=$PG_URI buildout:fe-ip=$ENCD_FE_IP buildout:remote_indexing=$ENCD_REMOTE_INDEXING buildout:index_procs=$ENCD_INDEX_PROCS buildout:index_chunk_size=$ENCD_INDEX_CHUNK_SIZE"
```

Comment:
This pattern is one reason we need to move to demo, test, rc, prod.cfgs.  I just had to tack on two variables for the offloaded indexers.  We should fix this soon since deployments are getting more complex.

Reply:
totally concur, I was wondering if we maybe even want 2 scripts.  One for deploy demo one for "production" deploy.  Maybe they could import a common module or something.